src/api/todos.js

```diff
@@ -0,0 +1,26 @@
+import { client } from '@/utils/httpClient'
+
+const userId = 1364

```

Comment:
The `userId` is hardcoded, which may not be suitable if the application needs to support multiple users. Consider passing `userId` as a parameter to the functions if dynamic user support is required.